### PR TITLE
[Communication] Update files for February release

### PR DESCRIPTION
**Pull request by JianpingChen**

### CODE REVIEW

File: sdk/communication/azure-communication-common/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0-beta.5 (2021-02-08)
+### Breaking Changes
+- Deleted CallingApplicationIdentifier
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-common/CHANGELOG.md:5]: CallingApplicationIdentifier is Removed

File: sdk/communication/azure-communication-common/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+- Deleted getId function from CommunicationIdentifier class
+
+### Added
+- New MicrosoftTeamsUserIdentifier constructor that takes a non-null CommunicationCloudEnvironment parameter
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-common/CHANGELOG.md:9]: Let's add "." at the end of them all

Comment (vcolin7) [reply sdk/communication/azure-communication-common/CHANGELOG.md:9]: I agree with Reza.

File: sdk/communication/azure-communication-common/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+- Added class CommunicationCloudEnvironment
+- Added class CommunicationCloudEnvironmentModel
+- Added class CommunicationIdentifierSerializer
+- Added class CommunicationIdentifierModel
+- Added class MicrosoftTeamsUserIdentifierModel
+- Added class PhoneNumberIdentifierModel
+- Added class CommunicationUserIdentifierModel
```
Comment (vcolin7) [inline sdk/communication/azure-communication-common/CHANGELOG.md:16]:
```suggestion
### New Features
- Added a new constructor to `MicrosoftTeamsUserIdentifier` that takes a non-null CommunicationCloudEnvironment parameter.
- Added class `CommunicationCloudEnvironment`.
- Added class `CommunicationCloudEnvironmentModel`.
- Added class `CommunicationIdentifierSerializer`.
- Added class `CommunicationIdentifierModel`.
- Added class `MicrosoftTeamsUserIdentifierModel`.
- Added class `PhoneNumberIdentifierModel`.
- Added class `CommunicationUserIdentifierModel`.
```

File: sdk/communication/azure-communication-common/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,19 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0-beta.5 (2021-02-08)
+### Breaking Changes
+- Deleted CallingApplicationIdentifier
+- Deleted getId function from CommunicationIdentifier class
```
Comment (vcolin7) [inline sdk/communication/azure-communication-common/CHANGELOG.md:6]:
```suggestion
### Breaking Changes
- Removed `CallingApplicationIdentifier`.
- Removed `getId` method from the `CommunicationIdentifier` class.
```

File: sdk/communication/azure-communication-common/CHANGELOG.md (hunk 3 above)
Comment (JianpingChen) [reply sdk/communication/azure-communication-common/CHANGELOG.md]: But another guideline says: "Added".

https://azure.github.io/azure-sdk/policies_releases.html#changelog-guidance

Comment (JianpingChen) [reply sdk/communication/azure-communication-common/CHANGELOG.md]: I'll ignore the changelog-guidance and follow the release notes guideline. :-)

Comment (vcolin7) [reply sdk/communication/azure-communication-common/CHANGELOG.md]: That's interesting, I need to talk to @czubair about this, I've seen a few libraries use the release notes guidelines instead, mostly due to automated scripts that picks up the changelog to populate each month's release notes.

**Review by vcolin7 — APPROVED** [review]

**Pull request merged**